Text/LaTeX/Base/Types.hs

```diff
@@ -26,21 +26,21 @@ type PackageName = String
 type PageStyle = String
 
 -- | Type of labels.
-newtype Label = Label String deriving (Eq, Show)
+newtype Label =
+    Label {
+      -- | Get the name of a label.
+      labelName :: String
+    } deriving (Eq, Show)
```

Comment:
This actually changed the Show instance.

Reply:
Ah, yes. Do you want me to revert this?

Reply:
We can either leave it like that and add a manually defined instance for Show or revert this part of the PR. If you want to do it, that's great! Otherwise I can do it too.

Thanks for your contributions by the way.